src/core/imageUtils.js

```diff
@@ -0,0 +1,10 @@
+/* global navigator, window */
+/* eslint-disable react/prop-types */
+
+import { isAllowedOrigin } from 'core/utils';
+import fallbackIcon from 'amo/img/icons/default-64.png';
+
+export function getIconUrl(addon) {
```

Comment:
This should be named `getAddonIconUrl` because I'm pretty sure we'll need to add one for themes later.